src/backend/dx11/src/lib.rs

```diff
@@ -678,7 +682,14 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
     {
         for cmd_buf in submission.cmd_buffers.into_iter() {
```

Comment:
note: `into_iter()` is called automatically by `for`, so you can remove it from here